Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:14.15.4-alpine3.10@sha256:fe215d05cdde4b7f2a0f546c88a8ddc4f5fa280a204acdfc2383afe901fd6d84 as build
+
+USER node
+
+WORKDIR /home/node
+
+COPY --chown=node packages/ .
+
+RUN npm --prefix frontend install frontend && \
+    npm --prefix frontend run build
+
+FROM node:14.15.4-alpine3.10@sha256:fe215d05cdde4b7f2a0f546c88a8ddc4f5fa280a204acdfc2383afe901fd6d84
+
+ENV NODE_ENV production
+
+RUN mkdir -p frontend/dist server
+
+COPY --from=build /home/node/frontend/dist frontend/dist
+COPY --chown=node packages/server server
+
+RUN npm --prefix server install server
+
+USER node
+
+WORKDIR /home/node/server
+
```

Comment:
In this form the app is not very graceful. It doesn't trap SIGTERM etc... Stopping it is not a pleasant experience. I would add tini as the entrypoint to fix this problem.
Like [this](https://github.com/krallin/tini).

Reply:
I did not yet succeed with this solution, but it is not the end of the world I guess

Reply:
Yep, you are right it didn't work.... I've done the changes and all is well